### ISSUE TITLE
Looking for info

### ISSUE THREAD

**Issue by rlust** [rvc-proxy/rvc-proxy#1]
I would like to know what "A computer with a canbus network card configured on interface can0" card you suggest?

I am very proficent with using ESP32 chips and programming.

This is great to see that work you have done!

**Comment by kellyfitzy** [comment]
I’m using a Raspberry Pi 3B+ and a Waveshare RS485 CAN HAT.

**Comment by cterwilliger** [comment]
I’m using a Raspberry Pi 3B+ and a PiCAN2 from CopperhillTech.

**Comment by rlust** [comment]
Thanks Guys for the reply!
Are you connecting to a RV? If so how is the connection made?
Trying to connect to a 2013 Entegra.

**Comment by kellyfitzy** [comment]
Yes, I'm connecting to a 2019 Tiffin Phaeton through a network port on the
Spyder system.

**Comment by kellyfitzy** [comment]
To clarify, a CAN network port, not ethernet.

**Comment by rlust** [comment]
I see a NET-Port connector on my Spyder unit. Where did you get the cable?

**Comment by rlust** [comment]
I have been able to get a pi to work on the 2013 Entegra. My next step is to run the rvc2mqtt.pl program. When I run it I get the following error.
Can't locate YAML/Tiny.pm in @INC (you may need to install the YAML::Tiny module) 

Looking for info on how to get the module installed/
Thank you.

**Comment by rlust** [comment]
Mike helped me out her is the answer:

cpanm is the perl module installer. You can install that with apt install cpanminus. Once it's installed, you can run cpanm install YAML::Tiny
